Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -0,0 +1,102 @@
+@page "/registration/{organisationId}/join-organisation"
+@using CO.CDP.OrganisationApp.Constants
+@model JoinOrganisationModel
+
+@{
+var organisationDetails = Model.OrganisationDetails!;
+var joinHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Join)).HasInnerHtml;
+}
+
+<a href="/registration/has-companies-house-number" class="govuk-back-link">
+    Back
+</a>
+
+<main class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <partial name="_ErrorSummary" model="@Model.ModelState" />
+            <fieldset class="govuk-fieldset">
+                <h1 class="govuk-heading-l">
+                    <span class="govuk-label--l">Join company: @organisationDetails.Name</span>
+                </h1>
+                <h2 class="govuk-heading-m">Organisation details</h2>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Organisation name
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">@organisationDetails.Name</p>
+                        </dd>
+                    </div>
+                </dl>
+                <br/>
+                <h2 class="govuk-heading-m">User details</h2>
+                <dl class="govuk-summary-list">
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Firstname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">John</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Lastname
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">Johnson</p>
+                        </dd>
+                    </div>
+                    <div class="govuk-summary-list__row">
+                        <dt class="govuk-summary-list__key">
+                            Email address
+                        </dt>
+                        <dd class="govuk-summary-list__value">
+                            <p class="govuk-body">john@johnson.com</p>
+                        </dd>
+                    </div>
+                </dl>
+            </fieldset>
+        </div>
+    </div>
+    <br/>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form method="post">
+                <fieldset class="govuk-fieldset" data-required="" data-error="Select yes if you want to join this organisation" aria-required="true">
```

Comment:
`data-required` and `data-error` do not do anything and can be removed. Same for `aria-required`. Saw that on a lot of fieldsets elsewhere and removed it recently, I think it's something that just been copied and pasted around